src/test/java/seedu/address/model/person/StudentTest.java

```diff
@@ -52,34 +55,34 @@ public void isSamePerson() {
     public void equals() {
         // same values -> returns true
         Student aliceCopy = new StudentBuilder(DANIEL).build();
-        assertTrue(DANIEL.equals(aliceCopy));
+        assertEquals(DANIEL, aliceCopy);
```

Comment:
Can help me change `aliceCopy` to `danielCopy` for the 2 usages here, missed that out in my initial commit 😅